TelCo.ColorCoder/Test.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Diagnostics;
```

Comment:
good categorization of "test code" into Test.cs.
You could have called the class also as Test.